src/main/java/com/bookJourney/springboot/dto/BookDetailsDTO.java

```diff
@@ -0,0 +1,27 @@
+package com.bookJourney.springboot.dto;
+
+import com.bookJourney.springboot.entity.BookStatus;
+import com.bookJourney.springboot.entity.EnumMood;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+
+@AllArgsConstructor
+@Data
+public class BookDetailsDTO {
```

Comment:
dlaczego nie record?

Reply:
@marzgra musialam dac class a nie record bo potrzebowałam metody "set" i nie wiem jak to z recordem zrobic :) IntelliJ wyswietlil ze record jest immutable